### DISCUSSION TITLE
Multiple Choice Binary Markets

### DISCUSSION THREAD

**Discussion by pwdel** [openpredictionmarkets/socialpredict#323] [Ideas]
We have an ongoing discussion regarding multiple choice binary markets.

https://github.com/orgs/openpredictionmarkets/projects/3/views/1?pane=issue&itemId=79801286

If you would like to be added to this project specifically, please ask here. Alternatively please put your comments here on this thread.

**Comment by astrosnat** [comment]
Hi! Please add me. Some initial thoughts:

I'm in favour of multiple choice markets. I have used them myself on Manifold, e.g. this multiple choice market for forecasting the mass of the neutrino: [https://manifold.markets/astroblob/what-will-katrin-determine-as-the-m](https://manifold.markets/astroblob/what-will-katrin-determine-as-the-m)

They're useful for figuring out specific ranges e.g. if you're trying to gauge a range of people coming to an event or something

I agree with your comments in issue #320, maybe we could make multiple choice markets more expensive, at least give admins an option to configure that in `setup.yaml` and then we can come back and see how it's worked out for admins deploying different instances. Also I would like for our UI to be faster and less clunky than Manifold's if possible. A major issue for me when betting on multiple choice markets is that on Manifold's Android app I find myself placing a bet and then it reloads the entire page...:(

Because Manifold is trying to incentivise its users to buy more mana, creating questions works out as more costly and less efficient than e.g. trying to maintain your daily prediction streak by betting on the daily coinflip. Basically the question creation is aimed at "whales". But as you say, we can configure it to make asking questions much cheaper for the user. And this is probably going to make more sense for e.g. corporate or school users who are focused on asking the right kinds of questions in the right way

**Reply by pwdel** [reply]
Ok, I think you have been added already @astrosnat but yes, thanks for your comment here too.

**Comment by wda58175-droid** [comment]
Hello, may I ask if it's possible to add Chinese characters?

**Reply by pwdel** [reply]
> Hello, may I ask if it's possible to add Chinese characters?

We have on the roadmap to add multi lingual support

**Reply by wda58175-droid** [reply]
Thank you very much for your reply

**Reply by wda58175-droid** [reply]
Hello, I would like to apply to join the test.
